packages/visibility_detector/test/widget_test.dart

```diff
@@ -57,11 +56,27 @@ void main() {
       expect(info.visibleFraction, 1.0);
 
       final bounds =
-          VisibilityDetectorController.instance.widgetBoundsFor(cellKey);
+      VisibilityDetectorController.instance.widgetBoundsFor(cellKey);
       expect(bounds, expectedRect);
     },
   );
 
+  _wrapTest(
+    'VisibilityDetector test with transform.scale',
+    callback: (tester) async {
+      final scaleButton = find.byKey(demo.scaleButtonKey);
+      await tester.tap(scaleButton);
+      await tester.pumpAndSettle();
+      await _doStateChange(tester, () { });
```

Comment:
I would replace these lines with:

```dart
      await tester.pump();
      await tester.pump(VisibilityDetectorController.instance.updateInterval);
```

Reply:
Thank you, I have changed that now, too.